data_management/autopreprocess_testing/preprocess_directory.py

```diff
@@ -19,12 +19,15 @@
 target_dir = os.getcwd() + '/processed_test_images/'
 
 # Get list of all image paths
-img_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
+jpg_files = glob.glob(source_dir + "/**/*.jpg", recursive=True)
+png_files = glob.glob(source_dir + "/**/*.png", recursive=True)
+img_files = jpg_files + png_files
+tif_files = glob.glob(source_dir + "/**/*.tif", recursive=True)
```

Comment:
move the tif_files line above the img_files
then add the tif_files to the img_files concat

Reply:
@jraval can you confirm that the preprocessing steps after this work with a `.tif` file? I've never tried it so just want to make sure.